rasa/core/policies/ted_policy.py

```diff
@@ -983,15 +984,17 @@ def load(
 
         Returns:
             Loaded policy
+
+        Raises:
+            PolicyModelNotFound if the model is not found in the supplied `path`.
```

Comment:
💯

Reply:
Can you please also remove the `Optional` from the return type?

Reply:
I just did before your comment 😄